### PR TITLE
 add user/post api

### PR DESCRIPTION
**Pull request by hewonjeong**
- add user/post api
- add test code for user/post api
- add router for user api

### CODE REVIEW

File: src/server/users/controllers.js [hunk 1]
```diff
@@ -1,9 +1,58 @@
...
-
+  post: (req, res, cb) => {
+    const currentTime = new Date();
+    const timeHash = KeyUtils.genTimeHash(currentTime);
+    const key = `${ENTITY.USER}-${timeHash}`;
+    const idxKey = KeyUtils.getPrefix(ENTITY.USER, STATE.ALIVE, timeHash);
+    const user = {
+      key: key,
+      name: req.body.name,
+      email: req.body.email,
```
Comment (festinalente91) [inline src/server/users/controllers.js:32]: is this only for the user who signup with email / pw ?

Comment (hewonjeong) [reply src/server/users/controllers.js:32]: I think we don't have to store user's account type or other Facebook information yet.
So I just store email & password without regarding to account type.

**Comment by ovekyc** [comment]
LGTM

**Comment by festinalente91** [comment]
LGTM

**Pull request merged**